Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Deleting-and-Restoring-Pages/index v7.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
The file name here is missing a `-` - that might cause some trouble.
Would you mind renaming the fie to `index-v7.md` ? 🙂

Reply:
done